src/core/build.gradle.kts

```diff
@@ -59,18 +59,20 @@ dependencies {
         because("XPathUtil: throws SaxonApiException")
     }
 
-    runtimeOnly("org.codehaus.groovy:groovy") {
-        because("Groovy is a default JSR232 engine")
+    runtimeOnly("org.apache.groovy:groovy") {
+        because("Groovy is a default JSR223 engine")
     }
-    arrayOf("dateutil", "datetime", "jmx", "json", "jsr223", "sql", "templates").forEach {
-        runtimeOnly("org.codehaus.groovy:groovy-$it") {
-            because("Groovy is a default JSR232 engine")
+    arrayOf("dateutil", "datetime", "jmx", "json", "jsr223", "sql", "templates","toml").forEach {
+        runtimeOnly("org.apache.groovy:groovy-$it") {
+            because("Groovy is a default JSR223 engine")
         }
     }
 
     implementation("com.fasterxml.jackson.core:jackson-annotations")
     implementation("com.fasterxml.jackson.core:jackson-core")
     implementation("com.fasterxml.jackson.core:jackson-databind")
+    implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-toml")
```

Comment:
Is it really needed for Groovy 4?

Reply:
@vlsi Thanks for reviewing. Groovy  4.0.0 has TOML support(incubating feature). I have also published a blog post about it. Please check [check](https://qainsights.com/working-with-toml-in-groovy-4-0-in-jmeter/).

Reply:
@vlsi any updates please.

Reply:
@vlsi Should I remove the jackson-dataformat or keep it? Please clarify.